third_party/proto/resourceusage/BUILD

```diff
@@ -1,6 +1,9 @@
 # This is a vendorised copy of Buildbarn's resource usage proto for display in the browser.
 proto_library(
     name = "resourceusage",
-    srcs = ["resourceusage.proto"],
+    srcs = [
+        "resourceusage.proto",
+        "resourceusage.pb.go",
```

Comment:
This doesn't seem right. The proto_library emits this file

Reply:
So, this is `puku fmt` that is doing this. I had reverted that change in a previous commit, and not sure _why_ Puku feels the need to do that.